### PR TITLE
Prices

### PR DESCRIPTION
**Pull request by gl2748**
Tentatively add steem-land price data.
Pending:

https://github.com/steemit/conveyor/pull/34#pullrequestreview-147476312

### CODE REVIEW

File: Sources/Steem/API.swift [hunk 1]
```diff
@@ -60,6 +60,18 @@ public struct API {
         public let method = "get_dynamic_global_properties"
         public init() {}
     }
+    
+    public struct SteemPrices: Decodable {
+        public let steemSbd: Int32
+        public let steemUsd: Int32
+        public let steemVest: Int32
```
Comment (jnordberg) [inline Sources/Steem/API.swift]: Prices are floating point

Comment (gl2748) [reply Sources/Steem/API.swift]: Ok Cool, and is this the right track for getting the new conveyor endpoints into the Wallet?

Comment (jnordberg) [reply Sources/Steem/API.swift]: Yeah for now at least, we’ll probably lift non steemd calls to the shared lib later

Comment (gl2748) [reply Sources/Steem/API.swift]: NVM, I see it's already hooked up there.
https://github.com/steemit/swift-steem/blob/b6e714c0054d0e0bd91c84073f50023c761647c7/Tests/SteemIntegrationTests/API.swift#L4-L8

**Comment by gl2748** [comment]
This will fail integration tests until conveyor has the Prices endpoint PR merged.

**Comment by roadscape** [comment]
Prices endpoint PR is ready to be merged

**Comment by roadscape** [comment]
Prices API was merged

**Pull request merged**